lib/id3c/cli/io/google.py

```diff
@@ -0,0 +1,81 @@
+"""
+Utilities for interacting with Google services.
+"""
+from enum import Enum
+from googleapiclient import discovery
+from googleapiclient.http import MediaIoBaseDownload
+from io import BytesIO
+from typing import Optional
+import re
+
+
+# See a full list of Google Drive MIME types at
+# https://developers.google.com/drive/api/v3/ref-export-formats
+
+class GoogleDriveExportFormat(Enum):
+    CSV = "CSV"
+    EXCEL = "Excel"
+    HTML = "HTML"
+    HTML_ZIPPED = "HTML zipped"
+    PDF = "PDF"
+    PLAIN_TEXT = "Plain text"
+    RICH_TEXT = "Rich text"
+    WORD = "Word"
+
+GOOGLE_FORMAT_MIME_MAP = {
+    GoogleDriveExportFormat.CSV: "text/csv",
+    GoogleDriveExportFormat.EXCEL: "application/vnd.openxmlformats-officedocument.spreadsheetml.sheet",
+    GoogleDriveExportFormat.HTML: "text/html",
+    GoogleDriveExportFormat.HTML_ZIPPED: "application/zip",
+    GoogleDriveExportFormat.PDF: "application/pdf",
+    GoogleDriveExportFormat.PLAIN_TEXT: "text/plain",
+    GoogleDriveExportFormat.RICH_TEXT: "application/rtf",
+    GoogleDriveExportFormat.WORD: "application/vnd.openxmlformats-officedocument.wordprocessingml.document"
+}
```

Comment:
Ok to leave them since they're here now, but as a general comment, it does seem to me like everything but the one we're using (Excel, and maybe CSV) are a YAGNI situation.

Reply:
I thought that if this is a general utility it would be nice to add the common formats.